components/math/multiplication.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+import { randomize } from "../../pages/api/questions/questionGenerator";
+import { getRandomItemFromArray } from "../../pages/api/random";
+
+export function getRndBlock() {
+  let x = randomize(1, 10);
+  let y = randomize(1, 10);
+  let z = x + y;
+  let blockArray = [x + " x " + y, z];
+  let block = getRandomItemFromArray(blockArray);
+  return block;
+}
+
+export default function MultiplicationGame() {
```

Comment:
The filename and this function name for the component should match.

Maybe call it MultiplicationBlock or MultiplicationSquare?